Modules/Core/Common/include/itkLaplacianOperator.h

```diff
@@ -81,28 +85,10 @@ class ITK_TEMPLATE_EXPORT LaplacianOperator : public NeighborhoodOperator<TPixel
     }
   }
 
-  /** Copy constructor   */
-  LaplacianOperator(const Self & other)
-    : NeighborhoodOperator<TPixel, VDimension, TAllocator>(other)
-  {
-    for (unsigned i = 0; i < VDimension; ++i)
-    {
-      m_DerivativeScalings[i] = other.m_DerivativeScalings[i];
-    }
-  }
-
   /** This function is called to create the operator  */
   void
   CreateOperator();
 
-  /** Assignment operator   */
-  Self &
-  operator=(const Self & other)
-  {
-    Superclass::operator=(other);
-    return *this;
```

Comment:
Looks like the old assignment of `LaplacianOperator` had a bug, as it forgot to copy `m_DerivativeScalings`!  😨

Reply:
For the record, the bug (missing assignment of `LaplacianOperator::m_DerivativeScalings`) appears to be introduced with revision 670af6e15e7f416cdd75cf8f61748fd3cda8a485, May 13, 2003, "ENH: Operator now takes image spacing into account", which added the data member without extending the `operator=`.